src/components/Footer/index.js

```diff
@@ -6,6 +6,7 @@ import trans from 'Src/i18n';
 
 import styled from '@emotion/styled';
 import moment from 'moment';
+import { useLang } from '../../utils/lang';
 
```

Comment:
@IntegralTeam not familiar with this fn, does it use the i18n-next implementation?

Reply:
Ok, I see it now in this PR.
Could you please elaborate why are you using it rather than i18n.$lang directly?

Reply:
Yep, that's a hook. Basically, react can't watch changes like vue, mobx or angular do. You need to add Statefull class component or functional component with useState hook to track the changes.

The useLang adds event listener to language change and triggers state change with updated language. 

If I'll not do that such way, the component won't know if language changed or not, that's the way react works.

Reply:
Btw, usually we don't refresh page by changing url component to ?lang=en or /en/anything, but switching it without page refresh. In that case we need to create special wrapper, that will be forced to re-render its contents after language switching.

Just telling you in case there's no other reasons to do it (like ability to share already localized links with users), it'll simplify lot of things if we remove language code from url.